R/initialization.R

```diff
@@ -30,9 +30,18 @@ get_inits_for_one_chain <- function(stan_data, stdev = 0.01) {
 
   init_list <- list(
     w = stats::rnorm(n_weeks - 1, 0, stdev),
-    m = stats::rnorm(1, 0, stdev * stan_data$m_stdev_prior),
-    m_r = stats::rnorm(1, 0, stdev),
-    m_first_obs = stats::rnorm(1, 0, 0.01),
+    offset_ref_log_r_t = stats::rnorm(
+      1, stan_data$offset_ref_log_r_t_prior_mean,
+      stdev
+    ),
```

Comment:
Sounds good. I would put the conditional in the `rnorm` call, i.e.

```R
    offset_ref_log_r_t = stats::rnorm(
      stan_data$n_subpops > 1, 
      stan_data$offset_ref_log_r_t_prior_mean,
      stdev
    ),
```
This should work because `FALSE` acts like zero and `TRUE` like one.

```R
> rnorm(2 > 1)
[1] 0.3041123
> rnorm(2 < 1)
numeric(0)
```

Reply:
> I am not sure how this passed CI 

website / vignette build only runs on PRs into `main` and `prod`, whereas this is a PR into different branch

https://github.com/CDCgov/ww-inference-model/blob/main/.github/workflows/pkgdown.yaml

Reply:
Ah this makes sense! Currently its sort of our end-to-end test... I will add a test for the hosp only version of the model!